src/devices/ExplorerHat/Lighting/Lights.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Linq;
+
+namespace Iot.Device.ExplorerHat.Lighting
+{
+    /// <summary>
+    /// Represents the Explorer HAT led array
+    /// </summary>
+    public class Lights : IDisposable
+    {
+        private const int LED1_PIN = 4;
+        private const int LED2_PIN = 17;
+        private const int LED3_PIN = 27;
+        private const int LED4_PIN = 5;
+
+        private GpioController _controller;
+
+        private List<Led> LedArray { get; set; }
+
+        /// <summary>
+        /// Blue led (#1)
+        /// </summary>
+        public Led One { get => LedArray[0]; }
+
+        /// <summary>
+        /// Yellow led (#2)
+        /// </summary>
+        public Led Two { get => LedArray[1]; }
+
+        /// <summary>
+        /// Red led (#3)
+        /// </summary>
+        public Led Three { get => LedArray[2]; }
+
+        /// <summary>
+        /// Green led (#4)
+        /// </summary>
+        public Led Four { get => LedArray[3]; }
```

Comment:
should this be `Color LedColor` instead? (haven't seen the device but guessing this is single RGB led)

Reply:
The hat has 4 leds, numbered from 1 to 4, and each one of a different color: blue (1), yellow (2), red (3), green (4). 

They are not rgb leds.

Reply:
fair enough